### PR TITLE
added content and proofreading

### PR DESCRIPTION
**Pull request by monikasmolarek**
I added some content - please review it for substantive correctness

### CODE REVIEW

File: docs/start/smesher/post_1n.md [hunk 1]
```diff
@@ -73,41 +73,35 @@ Run it with the following command:
...
-insert into poet_registration select * from srcDB.poet_registration;
-insert into nipost select * from srcDB.nipost;
-COMMIT;
-detach srcDB;
-```
-
-It is however recommended only for advanced users as it does not cover all the corner cases and may lead to data corruption.
-
-:::
+> [!NOTE]
```
Comment (pigmej) [inline docs/start/smesher/post_1n.md:76]: This is GH syntax. On that site we are using docusaurus which uses the syntax that was there :)

File: docs/start/smesher/post_1n.md [hunk 2]
```diff
@@ -73,41 +73,35 @@ Run it with the following command:
...
 - Detailed configuration steps can be found in the `post-service` [README](https://github.com/spacemeshos/post-rs/blob/main/service/README.md), guiding you through connecting each PoST service to your node.
 
 ## Operational Guide
 
-:::note
-
-This is just example, in real world you should use some kind of process manager like `systemd` or `supervisord` to manage your services. Or use some kind of orchestration tool like `kubernetes` or `docker-compose`.
-
-:::
+> [!WARNING]
```
Comment (pigmej) [inline docs/start/smesher/post_1n.md:103]: Same here :) it's GH syntax but we need a docusaurus instead;)

**Review by pigmej — CHANGES_REQUESTED** [review]
Mostly ok, the infos not ;)

File: docs/start/smesher/post_1n.md (hunk 2 above)
Comment (monikasmolarek) [reply docs/start/smesher/post_1n.md:103]: yep, I just figured it out and wanted to revert

File: docs/start/smesher/post_1n.md [hunk 3]
```diff
@@ -82,8 +82,8 @@ It is possible to merge nodes by hand too
 - Rename the key files respectively for easy identification of each identity.
 - Run: `sqlite3 target_node.sql` where `target_node.sql` is the database file of the node you're consolidating to.
 
-```attach '<source_path.sql>' as srcDB;
-BEGIN;
+ ```attach '<source_path.sql>' as srcDB;
+ BEGIN;
```
Comment (pigmej) [inline docs/start/smesher/post_1n.md:86]: I am not sure if I understand that change.

Comment (monikasmolarek) [reply docs/start/smesher/post_1n.md]: Neither do I. 🤦‍♀️

**Pull request merged**